astronomer/providers/apache/hive/sensors/hive_partition.py

```diff
@@ -0,0 +1,61 @@
+from typing import Any, Dict, Optional
+
+from airflow import AirflowException
+from airflow.providers.apache.hive.sensors.hive_partition import HivePartitionSensor
+
+from astronomer.providers.apache.hive.triggers.hive_partition import (
+    HivePartitionTrigger,
+)
+
+
+class HivePartitionSensorAsync(HivePartitionSensor):
+    """
+    Waits for a given partition to show up in Hive.
+    Note: HivePartitionSensorAsync uses implya library instead of pyhive
+    since pyhive is currently unsupported. Refer https://github.com/dropbox/PyHive.
+    Since we use implya library, please set the connection to use the port 10000 instead of 9083
+
+    :param table: the table where the partition is present.
+    :param partition: The partition clause to wait for. This is passed as
+        notation as in "ds='2015-01-01'"
+    :param schema: database which needs to be connected in hive. By default it is 'default'
+    :param metastore_conn_id: connection string to connect to hive.
+    :param polling_interval: The interval in seconds to wait between checks for partition.
+    """
+
+    def __init__(
+        self,
+        polling_interval: float = 5,
```

Comment:
are we doing async for https://github.com/apache/airflow/blob/main/airflow/providers/apache/hive/sensors/hive_partition.py#L27 
Then they have poke_interval instead of polling_interval and the default value is 60 * 3

Reply:
in that case we do not need this __init__ method

Reply:
removed the init as used default poke_interval